xcm/pallet-xcm/src/lib.rs

```diff
@@ -244,6 +244,12 @@ pub mod pallet {
 		/// The maximum number of local XCM locks that a single account may have.
 		type MaxLockers: Get<u32>;
 
+		/// The maximum number of users a single remote lock may have.
+		type MaxRemoteLockUsers: Get<u32>;
+
+		/// The ID type for remote lock users.
+		type RemoteLockIdentifier: Parameter + Member + MaxEncodedLen + Ord + Copy;
```

Comment:
Similar to `FreezeIdentifier` - https://github.com/paritytech/polkadot/blob/3ec08c0ac34d7c9c71e56e8d5c64151aa69c6d34/runtime/polkadot/src/lib.rs#L299
for example, pallet conviction_voting, can hold a lock on a remote lock.

Reply:
Ahh similar to the `HoldReason` or?

But then I'm even more confused, I first thought there would be one bigger remote lock and then users would be actual "users", but we probably just mean different "reasons" for locking or? Maybe it could be given the name `reason` or something similar for the field in `RemoteLockedFungibleRecord`?

Reply:
Discussed in DM, renaming `users` to `consumers`